pkg/cliutil/prepare/prepare.go

```diff
@@ -210,6 +224,7 @@ func CheckClusterPortConflict(clusterSpec *meta.SpecManager, clusterName string,
 			continue
 		}
 
+		// TODO
```

Comment:
Todo what?

Reply:
can not use the tidb cluster specified meta data
https://github.com/pingcap/tiup/pull/587/commits/f09b8b9491cebd0d73a25f0ecee68e857fff5b3d